BaseFilter/sections/specific.txt

```diff
@@ -15421,6 +15421,8 @@ movie-censorship.com##td[width^="160"][align="center"] > div[style*="border: 1px
 tubesex.me/eureka
 enworld.org##ul > li[class="postbitlegacy postbitim postcontainer"]
 pornhd.com##ul.thumbs + div.listing-zone.mobile-zone
+nowfoods.com##.block-sd-cookie-consent-block
```

Comment:
Cookies should be in `AnnoyancesFilter/cookies/cookies_specific.txt
Promo popup in `AnnoyancesFilter/popups/popups_specific.txt`

Reply:
Could you also place the rules at the beginning of the lists

Reply:
Fixed